haproxy/templates/configmap-ldap-tls.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ include "haproxy.fullname" . }}"
+  labels:
+    app.kubernetes.io/name: {{ include "haproxy.name" . }}
+    helm.sh/chart: {{ include "haproxy.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  haproxy.cfg: |-
+    global
+      log stdout local0
+      maxconn  2048
+      tune.bufsize  400000
+      tune.ssl.default-dh-param  2048
+
+    defaults
+      log global
+      option dontlognull
+      timeout connect 5s
+      timeout client 10s
+      timeout server 10s
+      timeout  check 15s
+      timeout  check 15s
+
+    frontend frontend
+      bind *:{{ .Values.haproxy.ldapTls.frontend.port }}
```

Comment:
We have to rework the whole port thing. When it is under ldapTls, it won't work for other configurations.